libevm/VMCalls.cpp

```diff
@@ -159,11 +167,13 @@ bool VM::caseCallSetup(CallParameters *callParams, bytesRef& o_output)
 {
 	m_runGas = toInt63(m_schedule->callGas);
 
-	if (m_OP == Instruction::CALL && !m_ext->exists(asAddress(m_SP[1])))
+	callParams->staticCall = (m_OP == Instruction::STATICCALL || m_ext->staticCall);
+
+	if ((m_OP == Instruction::CALL || m_OP == Instruction::STATICCALL) && !m_ext->exists(asAddress(m_SP[1])))
```

Comment:
Should we charge for new account creation in case of STATICCALL? @chriseth @vbuterin?